### PR TITLE
Add instructions on how to use the GitHub web editor to create translations to the CONTRIBUTING file

### PR DESCRIPTION
**Pull request by Steffo99**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,28 +8,86 @@ As a condition of providing a Contribution, you agree that:
...
-  * Con: the admins have to do the git-work for you, and as a result your changes will be logged as theirs.
-3. The professional way
-  * [Fork the repository](https://github.com/ReactiveDrop/reactivedrop_translations/fork), edit & commit it and then push your changes back to the origin.
-  * Pro: full control over big sets of changes, use functions like search & replace in files, regex, machine translation.
-  * Con: steeper learning curve using the git system.
-  * Use an editor to make your changes and integrate them through github. Read more about it [on our wiki](https://github.com/ReactiveDrop/reactivedrop_translations/wiki).
-
-## Pre-requisites
+
+* via the GitHub web editor [\[jump\]]()
```
Comment (BenLubar) [inline CONTRIBUTING.md:15]: are these meant to have anchors?

Comment (Steffo99) [reply CONTRIBUTING.md:15]: oops! adding them right now!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,28 +8,86 @@ As a condition of providing a Contribution, you agree that:
...
+
+## Using the GitHub web editor
+
+0. Ensure you are [logged in](https://github.com/login) to GitHub.
+
+1. Open the [GitHub web editor](https://github.dev/ReactiveDrop/reactivedrop_translations) by clicking on this link.
+
+2. When prompted, install the suggested extension.
+
+   ![Recommended Extensions tooltip, suggesting the installation of "Valve KeyValue File Support".](https://i.imgur.com/2YjF5ue.png)
```
Comment (BenLubar) [inline CONTRIBUTING.md:27]: would probably be better to upload the screenshots to a folder in this repo or to put them in developer.reactivedrop.com or something (not sure if imgur images are going to stay around; they do auto-delete images occasionally)

Comment (Steffo99) [reply CONTRIBUTING.md:27]: do they?  
I've never seen imgur deleting any image, and I can't find anything on the web about it

I agree that uploading them somewhere else would be a wise idea, but I'd avoid putting them in the repository, as that would make it a quite bit larger in file size...

Comment (BenLubar) [reply CONTRIBUTING.md:27]: I'd recommend developer.reactivedrop.com, then (it's a github pages site, so the repo just gets copied to a CDN).

I'm pretty sure I'm the only one with any reason to have developer.reactivedrop.com's repo downloaded right now, so no worries about people having to download the images repeatedly.

Comment (Steffo99) [reply CONTRIBUTING.md]: This should let us avoid going through GitHub Pages by using GitHub User Content as a CDN directly!

**Pull request merged**